keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -910,3 +910,5 @@ _This section must be completed when targeting beta graduation to a release._
 - 2018-01-29: Proposal submitted https://github.com/kubernetes/community/pull/1503
 - 2018-02-28: Alpha (v1.10) implemented https://github.com/kubernetes/kubernetes/pull/59495
 - 2018-06-04: Promoted to Beta (v1.11) https://github.com/kubernetes/kubernetes/pull/64482
+- 2021-05-11: Stable API approved https://github.com/kubernetes/enhancements/pull/2587
```

Comment:
Could you add details for when these beta API changes were made:

1. Install hint
2. Cluster info
3. Cluster config `client.authentication.k8s.io/exec`
4. Interactive mode
5. Metrics
6. Anything else you can think of

Reply:
446b2e44